app/views/comfy/admin/api_forms/_render.haml

```diff
@@ -64,7 +64,36 @@
         tags: $(this).attr('data-tags')
       })
       $(this).val(JSON.parse($(this).attr('default_value'))).change();
-    })
+    });
   });
 
+  // Disabling file attachment functionality
+  addEventListener('trix-initialize', function(e) {
+    $('.disable-trix-file-attachment').each(function(e) {
+      // Remove file-attach icon
+      $(this).find('trix-toolbar .trix-button-group--file-tools').remove();
+      // Disables functionality to drag-and-drop files onto the trix-editor
+      $(this).find('trix-editor').on('trix-file-accept', function(e) {
+        e.preventDefault();
+      });
+    });
+  });
+
+  function checkRequiredRichTextFields(e) {
+    let isRequiredFieldsFilled = true;
+
+    $('#api_form .required-rich-text-error').remove();
+
+    $('trix-editor[required]').each(function() {
+      const inputFieldId = $(this).attr('input');
+
+      if (!$(this).parent().find('input#' + inputFieldId).val()) {
```

Comment:
@alis-khadka lets extract this out into its own function in case we want to override it on the client side to show a different message. 

feel free to coordinate with @Ayon95 when negotiating the API surface

Reply:
@alis-khadka once this change is made, we can deploy to `staging`